src/LBPair.sol

```diff
@@ -117,6 +115,31 @@ contract LBPair is LBToken, ReentrancyGuardUpgradeable, Clone, ILBPair {
             protocolShare,
             maxVolatilityAccumulator
         );
+
+        string memory tokenXSymbol = IERC20Metadata(address(_tokenX())).symbol();
+        string memory tokenYSymbol = IERC20Metadata(address(_tokenY())).symbol();
+        string memory binStep = StringUtils.uint16ToString(_binStep());
+
+        _tokenName = string.concat("Liquidity Book Token ", tokenXSymbol, "/", tokenYSymbol, "/", binStep);
+        _tokenSymbol = string.concat("LBT_", tokenXSymbol, "/", tokenYSymbol, "/", binStep);
+
+        __ReentrancyGuard_init();
```

Comment:
this is superfluous because the non-reentrant modifier does the job https://github.com/DistributedCollective/sovryn-lb-dex/blob/364df5e/src/libraries/ReentrancyGuardUpgradeable.sol#L82